code/src/index.css

```diff
@@ -1,13 +1,207 @@
     sans-serif;
+
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  color: var(--fifth);
+
+  /* BACKGROUND ANIMATION */
+
+  background: linear-gradient(120deg, var(--primary), var(--fourth));
+  background-size: 500%;
+  animation-name: gradient;
+  animation-duration: 4s;
+  animation-iteration-count: infinite;
+  animation-direction: alternate;
+  /* https://www.youtube.com/watch?v=f3mwKLXpOLk */
+}
+
+@keyframes gradient {
+  from {
+    background-position: 0%;
+  }
+  to {
+    background-position: 100%;
+  }
+}
+
+.title {
+  font-family: "Helvetica Neue", sans-serif;
+  font-weight: 900;
+  font-size: 19px;
+  padding: 40px 0 0 5px;
+  margin: 0;
+}
+
+.sub-title {
+  margin: 5px 0 25px;
+  font-size: 14px;
+  padding-left: 5px;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.container {
+  margin: 30px auto 0;
+  width: 75%;
+}
+
+/**** FORM ****/
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-shadow: 7px 10px;
+  background-color: var(--secondary);
+  border: solid 1px var(--primary);
+  font-size: 12px;
+  color: var(--fifth);
+}
+.form {
+  width: auto;
+}
+
+.textarea-wrapper {
+  align-items: center;
+  justify-content: center;
+  width: auto;
+  font-family: "Courier New", Courier, monospace;
+  position: relative;
+}
+
+textarea {
+  width: 100%;
+  padding: 10px;
+}
+
+/* The the blue outline round textarea when focus will be hidden */
+textarea:focus {
+  outline-width: 0;
+}
+
+.counter {
+  position: absolute;
+  bottom: 0px;
+  right: 10px;
+  -ms-transform: translateY(-120%);
+  transform: translateY(-100%);
+  margin: 0;
+}
+
+/* Button */
+
+.button-wrapper {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+}
+
+.button-form {
+  background-color: var(--primary);
+  color: var(--sixth);
+  border-radius: 30px;
+  padding: 12px;
+  font-size: 15px;
+  margin: 20px;
+  border: none;
+}
+.button-form:hover {
+  /* background: var(--secondary); */
+  color: var(--secondary);
+}
+
+.button-form:active {
+  background-color: #f36f6f;
+}
+
+/**** TWEETS ****/
+
+.tweet-container {
+  margin: 20px 0;
+  padding: 20px 30px 20px;
+  box-shadow: 7px 10px;
+  background-color: var(--secondary);
+  border: solid 1px var(--primary);
+}
+
+.message-box {
+  overflow-wrap: break-word;
+}
+
+.like-wrapper {
+  display: flex;
+  align-items: center;
+}
+
+.like-time-box {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.button-likes,
+.button-no-likes {
+  background-color: var(--primary);
+  border-radius: 30px;
+  padding: 5px;
+  font-size: 15px;
+  margin-right: 10px;
+  width: 30px;
+  border: none;
+}
+
+.button-no-likes {
+  background-color: var(--fourth);
+}
+
+.heart-counter {
+  margin: 0;
+}
+
+.time-stamp {
+  justify-content: end;
+}
```

Comment:
Could be removed if cleanup. Alignment is done in `.like-time-box {justify-content: space-between;}`.

Reply:
🙏🏽 Thank you